contracts/interchain-token-service/InterchainTokenService.sol

```diff
@@ -533,28 +576,68 @@ contract InterchainTokenService is
     /**
      * @notice Executes operations based on the payload and selector.
      * @param sourceChain The chain where the transaction originates from
-     * @param sourceAddress The address where the transaction originates from
+     * @param sourceService The address of the remote ITS where the transaction originates from
      * @param payload The encoded data payload for the transaction
      */
     function execute(
         bytes32 commandId,
         string calldata sourceChain,
-        string calldata sourceAddress,
+        string calldata sourceService,
```

Comment:
Let's preserve the GMP API naming convention

Reply:
I agree